lang/cs/Org.Apache.REEF.Examples.HelloREEF/HelloREEFYarn.cs

```diff
@@ -115,6 +114,17 @@ private void LogApplicationReport()
         }
 
         /// <summary>
+        /// Kill Job Application.
+        /// </summary>
+        /// <param name="appId"></param>
+        private void KillJobApplication(string appId)
```

Comment:
We don't run .Net test on Yarn. To avoid polluting simple HelloREEF, this is called HellREEFYarn, that contains more Yarn related API calls. It also shows user an example for how to call those APIs.